### ISSUE TITLE
Create an icon for each of the types of grocers

### ISSUE THREAD

**Issue by cimmone** [codefordayton/grocer#5]
1 for each of the following:
Specialty
Farmer's Market
Convenience
Grocery
CSA

**Comment by LeahArmstrong** [comment]
I've created some possible icon solutions. What are some opinions on these?
![screen shot 2014-04-17 at 10 38 56 am](https://cloud.githubusercontent.com/assets/1608205/2732743/17c5ff98-c63e-11e3-83db-60f5dd0f1a8c.png)

**Comment by DavidEBest** [comment]
I like them. I think there is enough shape variety to scale well and be recognizable at small sizes.

**Comment by LeahArmstrong** [comment]
Missed the meeting tonight, sorry about that. As an update, I've attached some different variations of the icons. The top row (black and white) seem to be the most clear at small scale.

Since the icon design is complete, I'm going to close this and move to ticket #17 

![screen shot 2014-04-22 at 9 51 52 pm](https://cloud.githubusercontent.com/assets/1608205/2772890/1e6c4faa-ca8a-11e3-8bfc-a1446a608d7a.png)

**Comment by DavidEBest** [comment]
Those look great! I agree that the black outlined ones look the best at a small scale.

**Comment by dlangh** [comment]
Great icons!  I am going to re-open this issue for a short time until the icons are in-place on the website, which I will try and get done today and will then re-close.

**Comment by dlangh** [comment]
Ok, I didn't realize these images were all in one.  Was the assumption that we would use sprites and control which image is show via CSS?  If so that won't work with the leaflet api.  You pass in the image url as such:

return L.marker(latlng, {
            icon: L.icon({
                iconUrl: "assets/img/grocery.png",
                iconSize: [30, 40],
                iconAnchor: [12, 28],
                popupAnchor: [0, -25]
            }),
            title: feature.properties.name,
            riseOnHover: true
        });

So while I'm certainly generally in favor of sprites, in this case the easier approach will be to break them up into separate files.

**Comment by dlangh** [comment]
Dustin,

I cut an image out to see how it looks and I see that the background isn't transparent.  Can you break these up and make the backgrounds transparent?

Thanks!

Don

**Comment by LeahArmstrong** [comment]
@dlangh I referenced in the closing ticket that I'm moving on to ticket #17 which involves exporting all of the icons individually as SVGs, at which point they will be available. I just saved the icons out today, and I will work on implementing them into the structure. They will be separate from the background color marker in case they will be changed depending on the vendor type for some added visibility.
